internal/client/client.go

```diff
@@ -698,15 +698,15 @@ func (a *AuthApiClient) ListBackups() ybmclient.ApiListBackupsRequest {
 	return a.ApiClient.BackupApi.ListBackups(a.ctx, a.AccountID, a.ProjectID)
 }
 
-func (a *AuthApiClient) ListBackupPolicies(clusterId string, fetchOnlyActive bool) ybmclient.ApiListBackupSchedulesRequest {
+func (a *AuthApiClient) ListBackupPolicies(clusterId string, fetchOnlyActive bool) ybmclient.ApiListBackupSchedulesV2Request {
```

Comment:
Does this depend on the server side change making it to PROD?

Reply:
Since this is under FF, perhaps we should use a FF for CLI also. We sohould have two versions of the CLI based on the FF. Otherwise, this will break when run against prod/portal where FF is not enabled.

Reply:
Looks like it is already taken care of

Reply:
I've added an FF, but didn't use it for ListBackupPolicies. I will edit it. Thanks for pointing this out.